Misc/NEWS.d/next/Core and Builtins/2020-10-02-11-35-33.bpo-41894.ffmtOt.rst

```diff
@@ -0,0 +1,4 @@
+When loading a native module and a load failure occurs, prevent a possible
+UnicodeDecodeError when not running in a UTF-8 locale by decoding the load
+error message with ``PyUnicode_DecodeFSDefault`` instead of
```

Comment:
I think it is better to not mentioned PyUnicode_DecodeFSDefault and PyUnicode_FromString (which are implementation details and don't say much to common user) but just say that locale encoding is used instead of UTF-8.